test/tsconfig.json

```diff
@@ -1,18 +1,10 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "module": "commonjs",
-    "allowJs": true,
```

Comment:
for disabling comments like `@ts-ignore not TS yet` maybe enable js?

Reply:
We've decided not to allowJs because it has a lot of other implications and leaving them in gives us a goal to work towards; typescript-ifing everything so we can get rid of them.